### ISSUE TITLE
rip-pop

### ISSUE THREAD

**Issue by cldwalker** [defunkt/rip#109]
Added a rip-pop to complement to rip-push. Let me know if you want me to test this shell function and know how.
http://github.com/cldwalker/rip/tree/rip_pop

**Comment by josh** [comment]
cool

I don't think `rip-config` is stack aware. It looks like it clobbers any stacked envs.

It might be nicer to write rip-push/pop in ruby as much as possible to make it easier to test. Then do

```
function rip-pop {
  eval `rip-config --pop $1`;
};
```

**Comment by cldwalker** [comment]
I'll take a stab at printing push and pop in ruby. What if any error messages do we want for pushing envs that already pushed or don't exist and popping envs that don't exist or aren't pushed?

**Comment by josh** [comment]
Probably the same as `rip env`

```
$ rip env foo
Can't find ripenv `foo'
```

**Comment by cldwalker** [comment]
Updated [branch](http://github.com/cldwalker/rip/tree/rip_pop) to implement pop and push in ruby with tests and edge cases handled.

**Comment by cldwalker** [comment]
Merged at e65956159679472f829c1c6e96216f23fda04c9a
